superchain/configs/sepolia/arena-z.toml

```diff
@@ -0,0 +1,61 @@
+name = "Arena Z Testnet"
```

Comment:
We should remove this and related files before merging (keeping only the changes to the `validation` module).

Reply:
Note that having the file here triggers tests to run which are failing. So when we remove, CI should be green.